src/core/models.py

```diff
@@ -112,25 +136,50 @@ def chroot_acl(self) -> str:
             - 'w' - write
             - 'a' - append
         """
-        return self.relation_data.get("chroot-acl", "cdrwa")
+        return self.relation_data.get(
+            "chroot-acl", "cdrwa"
+        )  # pyright: ignore reportGeneralTypeIssues
```

Comment:
**todo:** Pyright ignore can be removed.

Reply:
All ignores removed from `model.py`